packages/flutter_tools/lib/src/fuchsia/fuchsia_pm.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../base/context.dart';
+import '../base/io.dart';
+import '../base/process_manager.dart';
+import '../globals.dart';
+
+import 'fuchsia_sdk.dart';
+
+/// The [FuchsiaSdk] instance.
+FuchsiaPM get fuchsiaPM => context.get<FuchsiaPM>();
+
+class FuchsiaPM {
```

Comment:
Would love some high level documentation explaining what each command does (even if it mostly points to existing pm documentation)

Reply:
Done.